model/model_training/configs/config.yaml

```diff
@@ -60,6 +60,11 @@ defaults:
   residual_dropout: 0.2
   use_flash_attention: true
   sort_by_length: false
+  use_system_prefix: false
+  system_prefix:
+    "You are Joi, a large language model trained by Open-Assistant. Answer as
+    concisely as possible.\nKnowledge cutoff: 2021-09-01\nCurrent date:
```

Comment:
I am against adding this to the default config without any validation. Adding a constant prefix for training will it likely make harder to evaluate the model because the "oa model" will be conditioned on the existing of this exact prefix.